fuel-core/src/database.rs

```diff
@@ -248,3 +260,132 @@ impl InterpreterStorage for Database {
         Ok(block.headers.producer)
     }
 }
+
+#[async_trait]
+impl RelayerDb for Database {
+    async fn get_validators(&self) -> HashMap<Address, u64> {
+        struct WrapAddress(pub Address);
+        impl From<Vec<u8>> for WrapAddress {
+            fn from(i: Vec<u8>) -> Self {
+                Self(Address::try_from(i.as_ref()).unwrap())
+            }
+        }
+        let mut out = HashMap::new();
+        for diff in self.iter_all::<WrapAddress, u64>(columns::VALIDATOR_SET, None, None, None) {
+            match diff {
+                Ok((address, stake)) => {
+                    out.insert(address.0, stake);
+                }
+                Err(_) => return out,
+            }
+        }
+        out
+    }
+
+    async fn get_validator_diffs(
+        &self,
+        from_fuel_block: u64,
+        to_fuel_block: Option<u64>,
```

Comment:
These say *_fuel_block here, while the `CurrentValidatorSet` which actually calls this api refers to these params as da_height.

Reply:
Yep leftover from old logic, will be updated in new pull request.